handlers/main.yml

```diff
@@ -21,5 +21,7 @@
       --destination {{ jekyll_build_root }}
       {{ (jekyll_build_baseurl | length > 0) |
          ternary('--baseurl ' + jekyll_build_baseurl, '') }}
+      {{ (jekyll_build_config | length > 0) |
+         ternary('--config ' + jekyll_build_config, '') }}
```

Comment:
Suggestion (not a blocker). Make `jekyll_build_config` a list of files and use the [`join` filter](http://docs.ansible.com/ansible/playbooks_filters.html#other-useful-filters)?